src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -349,14 +424,14 @@ bool CartGridStrategy::handleGcode(Gcode *gcode)
 
 // These are convenience defines to keep the code as close to the original as possible it also saves memory and flash
 // set the rectangle in which to probe
-#define LEFT_PROBE_BED_POSITION (0)
+#define LEFT_PROBE_BED_POSITION (x_start)
 #define RIGHT_PROBE_BED_POSITION (x_size)
 #define BACK_PROBE_BED_POSITION (y_size)
```

Comment:
They're usable as size because absolute is the same as relative when the start point is 0.

You need to ensure that your rectangle is using the values you think it is (effectively a find-replace). Since you shadowed the names I can't tell if the x_size used there is a local variable or the member variable.